R/survey_statistics_helpers.R

```diff
@@ -161,6 +161,7 @@ get_var_est_quantile <- function(stat, vartype, q, grps = "", level = 0.95, df =
 
   coef <- data.frame(matrix(coef(stat), ncol = out_width))
   names(coef) <- qnames
+  out <- lapply(out, as.data.frame)
```

Comment:
This was causing a test failure, apparently `bind_cols()` was not happy about additional `svyby` class. 

unless this is a `vctrs::` problem. ping @lionel-: 

``` r
library(vctrs)

df1 <- data.frame(x = 1)
df2 <- data.frame(y = 1)
class(df2) <- c("foo", "data.frame")
vec_cbind(df1, df2)
#> Error: Can't combine `..1` <data.frame<>> and `..2` <foo<>>.
```

<sup>Created on 2020-04-28 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Reply:
oops I may have forgotten to enable the df fallbacks in cbind, I'll look into it.

Reply:
We should probably add coercion methods for svyby though.

Reply:
Are you using an old version of vctrs? I'm seeing this:

```r
df1 <- data.frame(x = 1)
df2 <- data.frame(y = 1)
class(df2) <- c("foo", "data.frame")
vec_cbind(df1, df2)
#>   x y
#> 1 1 1
#> Warning message:
#> Can't combine <data.frame> and <foo>; falling back to <data.frame>.
#> ℹ Convert all inputs to the same class to avoid this warning.
#> ℹ See <https://vctrs.r-lib.org/reference/faq-warning-convert-inputs.html>.
```

This warning will probably become silent before the CRAN release.

Reply:
Thanks @lionel-. @gergness feel free to remove the `lapply()` call if you're ok with the warning.